### PR TITLE
Drop MySQL/MariaDB support.

### PR DESCRIPTION
**Pull request by bfabio**
Fixes #83.

### CODE REVIEW

File: env/tasks/main.yml [hunk 1]
```diff
@@ -3,5 +3,5 @@
 - name: Ensure that a SQL database is available
   assert:
     that:
-      - 'gitlab__database in [ "postgresql", "mariadb" ]'
+      - 'gitlab__database == "postgresql" ]'
```
Comment (drybjed) [inline env/tasks/main.yml]: You need to remove the leftover `]` character after the list was changed to a string.

**Comment by bfabio** [comment]
@drybjed This needs https://github.com/debops/debops-playbooks/pull/386 to pass the build.

**Comment by drybjed** [comment]
@bfabio Good call, actually the playbook in the test needs to pass first. I started working on it and had to release PostgreSQL roles in the process. Right now there seems to be an issue with the Go version. The current test is done on Ubuntu Precise; can you switch to Ubuntu Trusty in the `.travis.yml` file to see if that helps?

**Comment by drybjed** [comment]
@bfabio Never mind, I'm trying to switch to Trusty on Travis-CI in this PR: https://github.com/debops/ansible-gitlab/pull/97

So far the build for `gitlab-workhorse` fails. I tried to do it manually on Travis-CI to see what's wrong:
https://github.com/debops/test-suite/blob/master/ansible-gitlab/test#L85-L99

But the results aren't promising: https://travis-ci.org/debops/ansible-gitlab/builds/275767567#L2687-L2706

I'm not sure what's wrong there and why `go` cannot find these libraries... Any suggestions how to handle this? The build works fine on Debian Jessie...

**Comment by bfabio** [comment]
@drybjed I see `gitlab-workhorse v1.3.0` requires go 1.5+ while Trusty has got 1.2 and 1.3 if you add the backports. Could that be it?

What version ends up installed?

**Comment by drybjed** [comment]
It looks like it's currently Go 1.3.3. But that's a good point, I'll look into adding Go 1.5+ into Travis test for Trusty.

**Pull request merged**